### DISCUSSION TITLE
Is there a way to push event to client's on front-end after a delayed job has run?

### DISCUSSION THREAD

**Discussion by stephanoparaskeva** [OptimalBits/bull#1971] [Q&A] [answered]
I have a 30 minute delayed job to run, which processes data and makes a payment, saving the payment to postgres.
On completion of the job I need to push the payment success back to front-end. is this possible?

**Comment by manast** [comment, marked as answer]
sure, just use something like https://pusher.com/ when the job completes.

**Reply by stephanoparaskeva** [reply]
Thanks!
